docker-compose-with-certbot.yml

```diff
@@ -0,0 +1,88 @@
+version: "3.7"
+services:
+  cybercom_nginx:
+    image: nginx:1.17
+    expose:
+      - 80
+      - 443
+    ports:
+      - 80:80
+      - 443:443
+    depends_on:
+      - cybercom_api
+    volumes:
+      - ./dc_config/nginx/config/${NGINX_CONFIG}.conf:/etc/nginx/conf.d/default.conf
```

Comment:
Do we need the NGINX_CONFIG environment variable?

Reply:
@Nithivarn-Reddy, remove the two instances where the NGINX_CONFIG variable is set and commit your changes. Once they are updated, I will merge this PR.

Reply:
@tdpearson sorry for the delay, thought I updated. I have made the change. Please review it once. Thank you.